manifest.toml

```diff
@@ -55,8 +55,8 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://github.com/dokuwiki/dokuwiki/releases/download/release-2023-04-04a/dokuwiki-2023-04-04a.tgz"
-        sha256 = "153c99cf42b9068b1ec21a2c765b862a44b374ad2f1a39223f5511a982b160bb"
+        url = "https://github.com/dokuwiki/dokuwiki/archive/refs/tags/release-2023-04-04.tar.gz"
```

Comment:
Please revert this. release must be `2023-04-04a` (until a new security release)

I don't know which "download link" should be used

- https://github.com/dokuwiki/dokuwiki/releases/download/release-2023-04-04a/dokuwiki-2023-04-04a.tgz
  - from https://github.com/dokuwiki/dokuwiki/releases
- https://download.dokuwiki.org/src/dokuwiki/dokuwiki-2023-04-04a.tgz
  - from https://download.dokuwiki.org/archive